### PR TITLE
Fix/8157 remove cypress from documentation

### PR DESCRIPTION
**Pull request by ninjagirl888**
#### 📲 What

To remove all references to Cypress.

#### 🤔 Why
		
No longer supported by Stacks Test Framework
		
#### 🛠 How
		
Removal  files from repository
Updates to pages that reference Cypress

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR.
		 
#### 🕵️ How to test

Notes on how a reviewer can test the changes, e.g. how to run the tests.

#### ✅ Acceptance criteria Checklist

- [x] Code peer reviewed?
- [x] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [x] Passing any exploratory testing?
- [x] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: docs/testing/dotnet_testing_selenium.md [hunk 1]
```diff
@@ -212,29 +212,6 @@ One of the best methods to test your locator is working is by trying it in the b
 
 [Data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) can help identify the elements under test.
 
-### Example
```
Comment (JackMidd-Amido) [inline docs/testing/dotnet_testing_selenium.md:215]: I don't think this block should be deleted.
It references to Cypress, but it is actually for C# Selenium and therefore still relevant

File: docs/testing/testing_in_nx/frontend_testing_in_nx.md [hunk 2]
```diff
@@ -57,9 +56,7 @@ If you are considering adding visual regression tests to your application, then
     - A powerful visual regression testing tool that detects visual changes, updates of browsers or OS, and integrates with popular testing frameworks and CI/CD pipelines, making it easy to automate visual regression testing and improve the quality and user-friendliness of digital products. Our community of experts recommend this software as the best approach, to decide if it is right for you, please visit [Applitools Eyes!](https://applitools.com/platform/eyes/)
 - Native
     - A free alternative to cloud-based providers which enables developers to perform automated visual regression testing using the latest web platform features in their chosen testing framework.
-
-:::note
-
-We do not currently support visual testing with Ensono Stacks using our `@ensono-stacks/cypress` plugin
-
-:::
+<br></br>
```
Comment (JackMidd-Amido) [inline docs/testing/testing_in_nx/frontend_testing_in_nx.md:59]: Are these required? Can they be deleted?

File: asciidoc/testing/dotnet_testing_selenium.adoc [hunk 3]
```diff
@@ -219,30 +219,6 @@ One of the best methods to test your locator is working is by trying it in the b
 
 link:https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes[Data attributes] can help identify the elements under test.
 
-=== Example
```
Comment (JackMidd-Amido) [inline asciidoc/testing/dotnet_testing_selenium.adoc:222]: As per other comment, I think this should be retained as it is not cypress code, it is C# Selenium

File: asciidoc/testing/testing_in_nx/frontend_testing_in_nx.adoc [hunk 4]
```diff
@@ -56,4 +55,6 @@ If you are considering adding visual regression tests to your application, we pr
 * Native
   ** A free alternative to cloud-based providers that enables developers to perform automated visual regression testing using the latest web platform features in their chosen testing framework.
 
-NOTE: We do not currently support visual testing with Ensono Stacks using our link:../../getting_started/cypress/plugin-information.adoc[`@ensono-stacks/cypress`] plugin
+++++
```
Comment (JackMidd-Amido) [inline asciidoc/testing/testing_in_nx/frontend_testing_in_nx.adoc:58]: Is this required?

File: docs/testing/dotnet_testing_selenium.md (hunk 1 above)
Comment (ninjagirl888) [reply docs/testing/dotnet_testing_selenium.md:215]: Section restored

File: docs/testing/testing_in_nx/frontend_testing_in_nx.md (hunk 2 above)
Comment (ninjagirl888) [reply docs/testing/testing_in_nx/frontend_testing_in_nx.md]: Not really - i just put in to give a bit more space between the footer and the main page to make it look nice, but that is subjective.  I have removed the line breaks.
With
image
Without
Without
image

File: asciidoc/testing/dotnet_testing_selenium.adoc (hunk 3 above)
Comment (ninjagirl888) [reply asciidoc/testing/dotnet_testing_selenium.adoc]: Restored

File: asciidoc/testing/testing_in_nx/frontend_testing_in_nx.adoc (hunk 4 above)
Comment (ninjagirl888) [reply asciidoc/testing/testing_in_nx/frontend_testing_in_nx.adoc]: Again removed as per previous comment

**Review by JackMidd-Amido — APPROVED** [review]

**Review by Andy-J-marshall — COMMENTED** [review]
Looks good to me, but there's still 2 references to Cypress in the branch:

asciidoc/testing/dotnet_testing_selenium.adoc
docs/testing/dotnet_testing_selenium.md

Do we need to remove these references too?

Edit - this doesn't need deleting, see Jack's comment above

**Pull request merged**